src/translate.py

```diff
@@ -3755,6 +3755,12 @@ def process_instr(instr: Instruction) -> None:
             for slot in abi.arg_slots:
                 if slot.reg:
                     func_args.append(as_type(regs[slot.reg], slot.type, True))
+                elif slot.offset in subroutine_args:
+                    func_args.append(
+                        as_type(subroutine_args.pop(slot.offset), slot.type, True)
+                    )
+                else:
+                    func_args.append(ErrorExpr(f"Missing stack arg {slot.offset:#x}"))
```

Comment:
might be nice to extend the text to note that this is likely due to a m2c limitation